libs/community/tests/unit_tests/test_docai.py

```diff
@@ -23,3 +24,14 @@ def test_docai_parser_invalid_processor_name(processor_name: str) -> None:
     with patch("google.cloud.documentai.DocumentProcessorServiceClient"):
         with pytest.raises(ValueError):
             _ = DocAIParser(processor_name=processor_name, location="us")
+
+@pytest.mark.parametrize(
+    "processor_name",
+    [os.environ["PROCESSOR_NAME"]]
```

Comment:
this looks like integration test, please move there

and we'll need a terraform code to deploy the processor